### PR TITLE
Add an rpminfo command

### PR DESCRIPTION
**Pull request by icarus-sparry**
Based very much on example code from
github.com/cavaliercoder/go-rpm/cmd/rpminfo/main.go (BSD 3 clause
license).

Signed-off-by: Icarus Sparry <icarus.w.sparry@intel.com>

### CODE REVIEW

File: cmd/info.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+				return strings.Join(a, ", ")
+			},
+			"timestamp": func(t time.Time) rpm.Time {
+				return rpm.Time(t)
+			},
+		}).
+		Parse(tmpl)
+}
+
+func die(format string, a ...interface{}) {
```
Comment (matthewrsj) [inline cmd/info.go]: These would be good in an `internal/helpers` package when we start using them elsewhere (not needed now).

File: cmd/info.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+func init() {
+	rootCmd.AddCommand(rpminfoCmd)
+	rpminfoCmd.Flags().StringVarP(&queryFormat, "query", "f", defaultQueryFormat, "template for query")
+}
+
+// rpminfoCmd is the command to call rpminfo
+var rpminfoCmd = &cobra.Command{
+	Use:   "rpminfo",
+	Short: "Print info on rpm packages",
+	Long:  `???`,
```
[the remark this answers is not in the document]
Comment (icarus-sparry) [reply cmd/info.go]: Yes. Actually I was planning something rather more, namely the default template contents.
Somewhere (godoc?) I need to put the structure definition so people know what they can put into the template.

File: cmd/info.go [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+var queryFormat string
+
+func rpminfo(w io.Writer, args []string) {
+
+	qf, err := queryformat(queryFormat)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// debugging: fmt.Fprintf(w, "Using %v\n", queryFormat)
```
[the remark this answers is not in the document]
Comment (matthewrsj) [reply cmd/info.go]: how about defining a global `debug` variable and hiding this behind a `if debug {...}` guard?

**Comment by matthewrsj** [comment]
@icarus-sparry the commits that this PR relies on have been merged. Can you rebase against master and drop the duplicated commits from here so it can merge cleanly?

**Comment by icarus-sparry** [comment]
Closing in favor of #8

**Pull request closed without merging**